packages/shared/src/components/modals/VideoModal.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactElement } from 'react';
+import ReactModal from 'react-modal';
+import { Modal } from './common/Modal';
+import { ModalKind, ModalSize } from './common/types';
+import CloseButton from '../CloseButton';
+import { ButtonSize } from '../buttons/Button';
+import { LogoPosition } from '../Logo';
+
+interface VideoModalProps extends ReactModal.Props {
+  src: string;
+  title: string;
+}
+
+export default function VideoModal({
+  onRequestClose,
+  src,
+  title,
+  ...props
+}: VideoModalProps): ReactElement {
+  return (
+    <Modal
+      className="px-8 pepper-90"
```

Comment:
We don't have the popup on the Sketch understandably so as we are just reusing an old implementation, our modals have their own background color already, would it be not sufficient that we are applying a dark background on light mode as well?

Reply:
It does look exactly the same removing the pepper-90 there, so removing it in order for it to be defined by the modal itself :)